pandas/tests/frame/methods/test_sort_values.py

```diff
@@ -217,26 +217,71 @@ def test_sort_values_stable_descending_sort(self):
         sorted_df = df.sort_values(by="sort_col", kind="mergesort", ascending=False)
         tm.assert_frame_equal(df, sorted_df)
 
-    def test_sort_values_stable_descending_multicolumn_sort(self):
+    @pytest.mark.parametrize(
```

Comment:
if you want all combinations you can stack the parameters

Reply:
You mean parametrize over the two parameters individually? I'll have to build up the expected indices depending on the parameters, but sure, I can do that.

Reply:
I've added the individual parametrization. It makes it maybe a little more succinct. Was that what you meant @arw2019 ?

Reply:
yeah i think this looks good

i'd get rid of the `id`'s, we generally don't use those and it will make this way more concise when you can drop the `pytest.param`

Reply:
Perfect. I've removed the IDs. Thanks a lot for the quick help.